tests/vasp/lobster/schemas/test_lobster.py

```diff
@@ -165,3 +173,165 @@ def test_LobsterTaskDocument(lobster_test_dir):
     }
     gross_popp_here = doc2.gross_populations[10]["Mulliken GP"]
     assert expected_gross_popp == gross_popp_here
+
+
+def test_lobstertaskdocument_saved_jsons(lobster_test_dir):
+    """
+    Test if jsons are saved are valid
+    """
+    import os
+
+    from pymatgen.electronic_structure.cohp import Cohp, CompleteCohp
+    from pymatgen.electronic_structure.dos import LobsterCompleteDos
+
+    from atomate2.lobster.schemas import (
+        LobsterTaskDocument,
+        read_saved_json,
+    )
+
+    # Test cba saved jsons and reading
```

Comment:
Could you expand documentation and comments here?

Reply:
Done